### DISCUSSION TITLE
Welcome to switchboard.js Discussions!

### DISCUSSION THREAD

**Discussion by shadowmoose** [shadowmoose/switchboard.js#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DanChristos** [comment]
Hello! 
I've just come across this tool and I'm interested to use it, but am having some issues, I can get example/chat-swarm.html running no problem, but it is utilizing cdn, of course I would like to use it as a node.js module, so I installed via npm, I added `"type":"module"` to `package.json` to make import work. everything seems to go well until I initialize a `new Switchboard()` where I get an error related to CSPRNG, so I installed csprng via npm, but am still getting the same error. Any ideas where I'm going wrong here?

**Comment by shadowmoose** [comment]
Hey, unfortunately Switchboard is not currently built to run in Node - it uses several browser-specific APIs, and would need a WebRTC shim. I may eventually adjust it to support Node as well, but that's on the backburner until I can find the time. In the meantime, all contributions from anybody who wants to implement these polyfills are welcome. Sorry if this impacts your project.
